### DISCUSSION TITLE
What are the purposes of  prism-live-xxx.js

### DISCUSSION THREAD

**Discussion by cyfung1031** [PrismJS/live#21] [Q&A]
In the base prism, the languages are already included. (prism-css, prism-javascript, prism-markup)
The syntax are displayed correctly.

Even the following three .js files are not loaded, every things work fine. 
So what are the purposes of prism-live-xxx.js??

prism-live-css.js
prism-live-javascript.js
prism-live-markup.js

**Comment by LeaVerou** [comment]
Snippets, what do things like Cmd + / do, autocompletion, things like that.
